src/extras/primitives/primitives/a-text.js

```diff
@@ -0,0 +1,28 @@
+/* Experimental text primitive.
+ */
+var getMeshMixin = require('../getMeshMixin');
+var registerPrimitive = require('../primitives').registerPrimitive;
+var utils = require('../../../utils/');
+
+registerPrimitive('a-text', utils.extendDeep({}, getMeshMixin(), {
+  defaultComponents: {
+    'text': {anchor: 'align', width: 5}
+  },
+  mappings: {
+    align: 'text.align',
```

Comment:
sorry, parsing - are you asking to remove the defaults from being specified in the primitive because you want to make them rely on defaults from the mapped components themselves?